### PR TITLE
tighten sudoer access re: USB drive formatting

### PR DESCRIPTION
**Pull request by adghayes**
Remove unnecessarily broad `sudo` perms.

### CODE REVIEW

File: config/sudoers [hunk 1]
```diff
@@ -32,17 +32,11 @@ vx-admin ALL=(root:ALL) NOPASSWD: /vx/code/app-scripts/unmount-usb.sh
...
-vx-services ALL=(root:ALL) NOPASSWD: /vx/code/vxsuite/libs/usb-drive/src/unmount.sh
+vx-services ALL=(root:ALL) NOPASSWD: /vx/code/vxsuite/libs/usb-drive/scripts/*
 vx-ui ALL=(root:ALL) NOPASSWD: /vx/code/config/admin-functions/timedatectl
 vx-ui ALL=(root:ALL) NOPASSWD: /usr/bin/brightnessctl
 vx-ui ALL=(root:ALL) NOPASSWD: /usr/local/bin/tpm2-totp
 vx-ui ALL=(root:ALL) NOPASSWD: /vx/code/app-scripts/sign.sh
 vx-ui ALL=(root:ALL) NOPASSWD: /vx/code/app-scripts/mount-usb.sh
 vx-ui ALL=(root:ALL) NOPASSWD: /vx/code/app-scripts/unmount-usb.sh
-vx-ui ALL=(root:ALL) NOPASSWD: /vx/code/vxsuite/libs/usb-drive/src/mount.sh
-vx-ui ALL=(root:ALL) NOPASSWD: /vx/code/vxsuite/libs/usb-drive/src/unmount.sh
```
Comment (adghayes) [inline config/sudoers:44]: I don't think `vx-ui` needs these perms?

File: config/sudoers [hunk 2]
```diff
@@ -32,17 +32,11 @@ vx-admin ALL=(root:ALL) NOPASSWD: /vx/code/app-scripts/unmount-usb.sh
...
 vx-ui ALL=(root:ALL) NOPASSWD: /bin/efibootmgr
-vx-ui ALL=(root:ALL) NOPASSWD: /usr/sbin/sfdisk
-vx-ui ALL=(root:ALL) NOPASSWD: /usr/sbin/mkfs.fat
-vx-ui ALL=(root:ALL) NOPASSWD: /usr/sbin/mkfs.exfat
```
Comment (adghayes) [inline config/sudoers:48]: Closing the security gap.

File: config/sudoers [hunk 3]
```diff
@@ -32,17 +32,11 @@ vx-admin ALL=(root:ALL) NOPASSWD: /vx/code/app-scripts/unmount-usb.sh
 vx-admin ALL=(root:ALL) NOPASSWD: /vx/code/vxsuite/libs/auth/src/intermediate-scripts/create-cert
 vx-services ALL=(root:ALL) NOPASSWD: /vx/code/vxsuite/libs/auth/src/intermediate-scripts/create-cert
 vx-services ALL=(root:ALL) NOPASSWD: /vx/code/vxsuite/libs/auth/src/intermediate-scripts/sign-message
-vx-services ALL=(root:ALL) NOPASSWD: /vx/code/vxsuite/libs/usb-drive/src/mount.sh
-vx-services ALL=(root:ALL) NOPASSWD: /vx/code/vxsuite/libs/usb-drive/src/unmount.sh
+vx-services ALL=(root:ALL) NOPASSWD: /vx/code/vxsuite/libs/usb-drive/scripts/*
```
Comment (adghayes) [inline config/sudoers:35]: I don't know if we really need to enumerate the scripts here, or if this is OK.

**Review by benadida — APPROVED** [review]

File: config/sudoers (hunk 1 above)
Comment (adghayes) [reply config/sudoers:44]: @jonahkagan is there any particular reason this was added in?

Comment (jonahkagan) [reply config/sudoers:44]: I think I was just trying to mirror the existing permissions I saw in the file

**Pull request merged**